### ISSUE TITLE
Upgrading from 1.0.6 to 2.0.1 failed

### ISSUE THREAD

**Issue by vargylet** [aceberg/WatchYourLAN#122]
Hey,

I upgraded from 1.0.6 to 2.0.1 and the history got cleared (according to the logs) and I got an error as well, but I can still access the GUI. The GUI is responding, but only showing an empty frontend.

docker-compose.yml
```
---
services:
  wyl:
    image: aceberg/watchyourlan:2.0.1
    container_name: watchyourlan
    network_mode: "host"
    volumes:
    - ${DATA}:/data
    environment:
      TZ: ${TZ}
      IFACE: ${INTERFACES}
      GUIPORT: "8840"
      TIMEOUT: "120"
      SHOUTRRR_URL: "slack://${SLACK_TOKEN}@${SLACK_CHANNEL}?color=warning"
      HISTORY_DAYS: "14"
      THEME: "darkly"
      COLOR: "dark"
    restart: always
    labels:
      - wud.tag.include=^\d+\.\d+\.\d+$
```

config.yaml
```
/data # cat config.yaml 
arp_timeout: "500"
auth: true
auth_expire: 7d
auth_password: [redacted]
auth_user: [redacted]
color: dark
dbpath: /data/db.sqlite
guiip: 0.0.0.0
guiport: "8840"
history_days: "14"
iface: eth0 net1 net2
ignoreip: "no"
loglevel: verbose
shoutrrr_url: slack://[redacted]?color=warning
theme: darkly
timeout: 120
```

Docker logs
```
$ docker logs watchyourlan 
2024/09/05 12:30:03 INFO ARP_STRS_JOINED: 
2024/09/05 12:30:03 INFO Log level=INFO
2024/09/05 12:30:03 INFO Using DB type=sqlite
2024/09/05 12:30:03 INFO Config dir path=/data/WatchYourLAN
2024/09/05 12:30:03 INFO =================================== 
2024/09/05 12:30:03 INFO Web GUI at http://0.0.0.0:8840
2024/09/05 12:30:03 INFO =================================== 
2024/09/05 12:30:03 INFO Removing all History before date="2024-09-03 12:30:03"
2024/09/05 12:30:03 INFO Removed records from History n=0
2024/09/05 12:30:03 ERROR exit status 1
```

I'm happy to share any additional information if that can help 😊

**Comment by sbe-arg** [comment]
Not sure there is auth in v2...

**Comment by luckman212** [comment]
Right. There's no auth in there. I just searched the codebase.

**Comment by vargylet** [comment]
Thanks for your comments! They lead me on the right path. I missinterpreted the note around v1 vs. v2 and thought that it was possible to upgrade from v1 to v2 and keep the history. I installed a separate instance of v2 and reviewed all the environment variables since some of them have changed. When I only used the correct environment variables v2 works like a charm.

Thank you! I hope this can info can help someone else 😊
